packages/safety-suite-api/src/api/dashboardSink/types.ts

```diff
@@ -1,3 +1,5 @@
+import {AggregateFunctionLiteral} from '../models';
```

Comment:
We should make our own type instead of re-using the query API's set.  They will start to diverge at some point.